src/components/pages/TransactionHistoryPage.js

```diff
@@ -1,34 +1,106 @@
-import { React, useState, useEffect } from "react";
+import { React, useState, useEffect, useContext, useCallback } from "react";
 import Navbar from "../reusable/NavBar";
 import Sidebar from "../reusable/SideBar";
+import AlgoquantApiContext from "../../api/ApiContext";
+import Table from "../reusable/Table";
+import formatter from "../utils/CurrencyFormatter";
+import { TableSpinner } from "../reusable/LoadSpinner";
 
+const FETCH_AMOUNT = 10;
 const TransactionHistoryPage = () => {
+  const [history, setHistory] = useState([]);
+  const [lastPage, setLastPage] = useState(false);
   const [transactions, setTransactions] = useState([]);
   const [page, setPage] = useState(1);
+  // State variables used to access algoquant SDK API and display/ keep state of user data from database
+  const algoquantApi = useContext(AlgoquantApiContext);
+  const [lastKey, setLastKey] = useState(null);
+  const [pagesSeen, setPagesSeen] = useState(0);
+  const [isLoading, setIsLoading] = useState(true);
+
+  const fetchTrades = useCallback(
+    (FETCH_AMOUNT) => {
```

Comment:
Is FETCH_AMOUNT needed as a parameters to this callback ? Since it is a constant I don't think it would need to be passed in each time.

Reply:
removed